pkg/apis/cluster/types.go

```diff
@@ -0,0 +1,116 @@
+package cluster
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+//revive:disable:exported
+
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// Cluster represents the desire state and status of a member cluster.
+type Cluster struct {
```

Comment:
Where is the json annotations of this struct? Like `json:"status,omitempty"`?

Reply:
It is a internal type of `Cluster`, refer to:

https://github.com/kubernetes/kubernetes/blob/489fb9bee3f626b3eeb120a5af89ad8c2b2f1c20/pkg/apis/apps/types.go#L315-L318

Reply:
I asked the same question too. :)